katago_server/games/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from katago_server.games.models import SelfPlay, Match
+
+
+@admin.register(SelfPlay)
```

Comment:
Can you explain what the relationship is between the SelfPlay and SelfPlayAdmin?

Reply:
Selfplay is the model. It generate sql base on an orm, and create a table in db (simplistic explanation)

SelplayAdmin is an admin object. 

When saying 

@admin.register(model)

We add a new section in django admin (see discord for credentials)

Reply:
We then can customize the admin itself by using the class under.

Reply:
See https://djangobook.com/mdj2-django-admin/ for more details / example.